docker/playground-ams.Dockerfile

```diff
@@ -13,6 +13,8 @@ ENV ARCTIC_HOME=/opt/arctic
 ENV HADOOP_CONF_DIR=/etc/hadoop/conf
 
 RUN set -x && \
+    apt-get update && \
+    apt-get install -y mysql-client && \
```

Comment:
it's fine to move it to playground-base

Reply:
Won't this make all images larger?

Reply:
Yes, and other upper layer image can benefit by this. ideally, we should add this wait logic to any image that depends on mysql

Reply:
It does not increase all images size. Since the image is layered, and the layer can be shared across images.